cdap-docs/reference-manual/source/http-restful-api/lifecycle.rst

```diff
@@ -25,6 +25,10 @@ with the name of the JAR file as a header::
 
   X-Archive-Name: <JAR filename>
 
+configuration object as a serialized json as another header (optional) ::
+
+  X-App-Config: <Serialized JSON of the Configuration Object>
```

Comment:
I think we need more detail about what this means exactly, probably with an example to illustrate